package.json

```diff
@@ -93,7 +93,8 @@
     "benchmark": "^2.1.4",
     "mkdirp": "^1.0.4",
     "protons": "^4.0.0",
-    "sinon": "^14.0.0"
+    "sinon": "^14.0.0",
+    "typescript": "^4.7.4"
```

Comment:
`typescript` is not used by this repo directly, it's used by `aegir` which has it as a dependency so this shouldn't be necessary.

Are you using yarn?

Reply:
I am using yarn. I was getting errors before this, and my hypothesis was that aegir was picking up my /usr/bin/tsc . 

I can remove this.

Reply:
A dep that's a dep of aegir depends on an old ts version.  For some reason yarn chooses to symlink that one in `node_modules/.bin`, rather than the one aegir depends on, even though it's lower in the dep tree.

npm chooses the correct version to symlink 🤷